boards/b-l072z-lrwan1/include/periph_conf.h

```diff
@@ -159,6 +159,7 @@ static const spi_conf_t spi_config[] = {
 #define I2C_NUMOF           I2C_0_EN
 #define I2C_IRQ_PRIO        1
 #define I2C_APBCLK          (CLOCK_APB1)
+#define I2C_SPEED           I2C_SPEED_NORMAL
```

Comment:
I think it would be great to take advantage of the refactoring to get rid of this "old-style" define way and use an array of struct as it's done for other periph.